package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "desk_allocation",
+  "version": "0.0.1",
+  "description": "",
+  "main": "run.py",
+  "scripts": {
+    "build": "webpack --mode production",
+    "start": "npm run build && python run.py",
+    "client": "webpack-dev-server --mode development --devtool inline-source-map --hot",
+    "server": "python run.py",
+    "dev": "concurrently \"npm run server\" \"npm run client\"",
+    "postinstall": "npm build"
+  },
+  "author": "José Ribeiro",
+  "dependencies": {
+    "babel-polyfill": "^6.26.0",
+    "prop-types": "^15.7.2",
+    "react": "^16.5.2",
+    "react-dom": "^16.5.2"
+  },
+  "devDependencies": {
+    "@babel/core": "^7.0.0",
+    "@babel/plugin-proposal-class-properties": "^7.0.0",
+    "@babel/preset-env": "^7.0.0",
+    "@babel/preset-react": "^7.0.0",
+    "babel-eslint": "^10.0.0",
+    "babel-loader": "^8.0.0",
+    "clean-webpack-plugin": "^1.0.0",
+    "concurrently": "^4.0.0",
+    "css-loader": "^2.0.0",
+    "eslint": "^5.0.0",
+    "eslint-config-airbnb": "^17.0.0",
+    "eslint-plugin-import": "^2.11.0",
+    "eslint-plugin-jsx-a11y": "^6.0.3",
+    "eslint-plugin-react": "^7.7.0",
+    "file-loader": "^3.0.0",
+    "html-webpack-plugin": "^3.2.0",
+    "style-loader": "^0.23.0",
+    "url-loader": "^1.0.1",
+    "webpack": "^4.5.0",
```

Comment:
shouldn't webpack be in normal dependencies? For the build script

Reply:
No, it is only used for bundling the client app, but it's not used by the app in production.  There's a _very_ discussion on this [here](https://github.com/webpack/webpack/issues/520)